src/server/index_patterns/service/lib/field_capabilities/field_caps_response.js

```diff
@@ -63,27 +64,45 @@ export function readFieldCapsResponse(fieldCapsResponse) {
     const capsByType = capsByNameThenType[fieldName];
     const types = Object.keys(capsByType);
 
+    // If there are multiple types but they all resolve to the same kibana type
+    // ignore the conflict and carry on (my wayward son)
     if (types.length > 1) {
-      return {
-        name: fieldName,
-        type: 'conflict',
-        searchable: false,
-        aggregatable: false,
-        readFromDocValues: false,
-        conflictDescriptions: types.reduce((acc, esType) => ({
-          ...acc,
-          [esType]: capsByType[esType].indices
-        }), {})
-      };
+      const uniqueKibanaTypes = uniq(types.map(castEsToKbnFieldTypeName));
+      if (uniqueKibanaTypes.length > 1) {
+        // If a single type is marked as searchable or aggregatable, all the types are searcuable or aggregatable
+        const conflictIsSearchable = types.some(type => {
+          return !!capsByType[type].searchable ||
```

Comment:
This logic would probably make a nice little helper function that could be reused below too